### PR TITLE
Announcing Scala.js 0.6.8.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki and @gzm0, for announcement on Friday morning CET, assuming everything goes well in the release process.

File: assets/badges/scalajs-0.6.8.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="98" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="98" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h51v20H0z"/><path fill="#007ec6" d="M51 0h47v20H51z"/><path fill="url(#b)" d="M0 0h98v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="25.5" y="15" fill="#010101" fill-opacity=".3">scala.js</text><text x="25.5" y="14">scala.js</text><text x="73.5" y="15" fill="#010101" fill-opacity=".3">0.6.8+</text><text x="73.5" y="14">0.6.8+</text></g></svg>
```
Comment (gzm0) [inline assets/badges/scalajs-0.6.8.svg]: No EOL at EOF intentional?

**Comment by gzm0** [comment]
LGTM otherwise.

File: assets/badges/scalajs-0.6.8.svg (hunk 1 above)
Comment (sjrd) [inline assets/badges/scalajs-0.6.8.svg]: Yes.  I downloaded the file as is. Our other svgs don't have EOLs either.

**Comment by nicolasstucki** [comment]
LGTM

**Pull request merged**